packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,206 @@
+- *Methods* - None
+- *Events* - None
+- *CSS Classes and CSS Custom Properties that affect the component* - None
+- *Slots*
+    - *default*
+
+### Anatomy
+
+#### Anchor Tabs
+
+```html
+<template>
+    <slot name="start"></slot>
+    <nav class="tablist">
+        <ul>
+            <slot name="anchortab"></slot>
+        </ul>
+    </nav>
+    <slot name="end"></slot>
+</template>
+```
+
+#### Anchor Tab
+
+```html
+<template slot="anchortab">
+    <li>
+        <a>
+            <slot></slot>
+        </a>
+    </li>
+</template>
+```
+
+### Angular integration
+
+There will be Angular directives for both `nimble-anchor-tabs` and `nimble-anchor-tab`. The latter will derive from `NimbleAnchorBaseDirective` which has the standard HTML anchor attributes.
+
+Tabs do not participate in forms, so there is no ControlValueAccessor.
+
+**[routerLink] and [routerLinkActive] Support**
+
+We will follow the same approach previously used for the BreadcrumbItem, Anchor, and AnchorButton:
+
+-   We will create a directive that extends `RouterLinkWithHref`, and its selector will be `nimble-anchor-tab[nimbleRouterLink]`. The directive will define `@Input nimbleRouterLink` (which sets `routerLink`). This is a small change for clients which we will document (consistent with BreadcrumbItem, Anchor, and AnchorButton), and other routerLink attributes can still be used as-is:
+    ```html
+    <nimble-anchor-tab
+        nimbleRouterLink="/customapp"
+        [queryParams]="{debug: true}"
+        [state]="{tracingId: 123}"
+    >
+        Custom App Page
+    </nimble-anchor-tab>
+    ```
+-   A separate directive will be created to disable use of the `routerLink` attribute on `nimble-anchor-tab` elements.
+
+### Blazor integration
+
+Blazor wrappers will be created for both `nimble-anchor-tabs` and `nimble-anchor-tab`. There are no special routing considerations/mechanisms for Blazor.
+
+### Visual Appearance
+
+[XD Design](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b2aa2c0c-03b7-4571-8e0d-de88baf0814b/)
+
+The design for navigation tabs will be the same as for standard tabs. The standard Nimble tabs do not (yet) have specific support for icons or alert indicators, so we will (initially) not implement those for navigation tabs either.
+
+In addition to indicating which tab is active, navigation tabs must also separately indicate which tab has keyboard focus. I.e. you can arrow through tabs without activating those tabs, and there must be a visual indication of which tab has keyboard focus. We will indicate the focused tab with a double underline similar to what the existing tab control shows when keyboard focused, but it will be two thin lines when focusing a non-selected tab. When focusing the selected tab, the two thin underlines will overlap the thicker underline indicating selection, so we end up with a visual that is consistent with our existing tab control.
+
+---
+
```

Comment:
Do you know yet whether we will be sharing any logic or styling code between the original tab implementation and the anchor tabs? We don't necessarily need to know the answer before approving the spec, but given the back-and-forth we had about sharing code between buttons and anchor buttons, it would be good to get everyone's buy-in on the plan before getting too far with implementation.

Reply:
I hope we can, but it's hard to tell without actually getting to the implementation. There are going to be changes related to handling selection and focus separately, but I think we can still use the modified CSS for the standard tabs. I've added a note about this to the spec.